docs/ref/contrib/admin/index.txt

```diff
@@ -1615,8 +1615,21 @@ templates used by the :class:`ModelAdmin` views:
                     kwargs["queryset"] = Car.objects.filter(owner=request.user)
                 return super(MyModelAdmin, self).formfield_for_foreignkey(db_field, request, **kwargs)
 
-    This uses the ``HttpRequest`` instance to filter the ``Car`` foreign key
-    field to only display the cars owned by the ``User`` instance.
+    This uses the :class:`~django.http.HttpRequest` instance to filter the
```

Comment:
I guess it'd be more consistent to link every instance of HttpRequest in this document, although I'm not convinced it adds much.